ci/functions.sh

```diff
@@ -4,7 +4,7 @@ DEFAULT_CAS_VERSION=6.4.0-SNAPSHOT
 DEFAULT_BOOT_VERSION=2.4.4
 
 function downloadTomcat() {
-  tomcatVersion="9.0.45"
+  source ./gradle.properties
```

Comment:
This is OK, but I think it would be better if you did something like:

```
tomcatVersion=$(cat gradle.properties | grep tomcatVersion | awk -F"=" '{printf $2}')
```

Reply:
I made a getProperty function but it took me way longer than it should have to realize it wasn't working because the working directory was changing before downloadTomcat was called and I was trying to use relative path for gradle.properties.